### ISSUE TITLE
Plugin calculates wrong max-height

### ISSUE THREAD

**Issue by julmot** [gromo/jquery.scrollbar#63]
I am using the plugin as a body-scrollbar. Unfortuantely it calculates a wrong max-height. The wrong value is exactly the browsers scrollbar width/height. So for example if my viewport is 687px height it will set the max-height of the scrollbar-div to 704px. What could be the reason? Currently I can not scroll to the bottom:
![max](https://cloud.githubusercontent.com/assets/8310677/9936667/69997b76-5d5d-11e5-8970-6a9658a7ccb0.png)

**Comment by julmot** [comment]
@gromo

**Comment by gromo** [comment]
Hi.

Sorry for delay, but I'm extremely busy these days - working on urgent project. Can you prepare example on jsFiddle.net or send me email with attachment? I will try to look at it at the nearest time, but I think it will be at the end of the next week.

**Comment by julmot** [comment]
Unfortunately I can't show it to you. I created the [fiddle](http://jsfiddle.net/7ww3zgob/1/). However I found the problem. I have used `overflow-x:hidden !important` which the plugin can't handle. When removing it the plugin calculates the correct max-height. However, then I have this problem: https://github.com/gromo/jquery.scrollbar/issues/61
The x-scrollbar from the browser will appear if the content is not height enough. This is why I had to add `overflow-x:hidden !important`.
